WreckEngine/Game.h

```diff
@@ -14,16 +14,18 @@
 class Game
 {
 public:
-	Game() = default;
-	Game(GLprogram prog);
+    Game();
```

Comment:
not before, but now it effectively is. Might as well, it's either nullptr or uninitialized now.